sqldb/sqlc/queries/graph.sql

```diff
@@ -17,7 +17,9 @@ ON CONFLICT (pub_key, version)
         last_update = EXCLUDED.last_update,
         color = EXCLUDED.color,
         signature = EXCLUDED.signature
-WHERE EXCLUDED.last_update > nodes.last_update
+WHERE EXCLUDED.last_update IS NULL
```

Comment:
nit: i think it's safe to remove `EXCLUDED.last_update IS NULL` as having that would allow overwriting an existing nodes row if the node we're inserting has NULL `last_update`.

Reply:
ah indeed - thanks 🙏  (although we will probs add it back when we introduce v2 since then we wont have last_update set. But yeah, for now safe to delete!)